ddmrp/models/stock_buffer.py

```diff
@@ -1341,7 +1349,7 @@ def _search_open_stock_moves_domain(self):
                 "in",
                 ["draft", "waiting", "confirmed", "partially_available", "assigned"],
             ),
-            ("location_dest_id", "=", self.location_id.id),
+            ("location_dest_id", "child_of", [self.location_id.id]),
```

Comment:
can you implement this as it was done here https://github.com/OCA/ddmrp/commit/6c060adaf1fa0bd56a2f54e7d57eff9c1e9508ea ?

Reply:
Done!